src/app/geo/map/leaflet-map/leaflet-config.ts

```diff
@@ -5,26 +5,28 @@ export class LeafletConfig {
   static INIT_ZOOM = 14;
   static CONTROL_ZOOM_POSITION: ZoomOptions = { position: 'topright' };
   private static MARKERS_PATH = '../assets/leaflet-icons/';
```

Comment:
This relative path was responsible of markers not being loaded in a previous PR. Just FYI. And bear in mind that the trailing slash makes _iconUrl_ a malformed url in line 15.  I cannot see if the trailing slash is needed for any reason. If not it would be better to remove it and add slashes in lines 18 and 21.